event-processing/event-processing-template.yml

```diff
@@ -1898,6 +2035,7 @@ Resources:
           - AUTH_UPDATE_PROFILE_PHONE_NUMBER
           - AUTH_UPDATE_PROFILE_CONSENT_UPDATED
           - AUTH_UPDATE_PROFILE_TERMS_CONDS_ACCEPTANCE
+          - AUTH_PROFILE_AUTH_APP
```

Comment:
I counted 56 events implemented in the spreadsheet but only 48 here. If there are missing events, I am happy for  that to be in a separate PR.